modules/bindgen/src/main/scala/analysis/visitStruct.scala

```diff
@@ -111,6 +111,24 @@ def visitStruct(cursor: CXCursor, name: String)(using
                   fieldName -> constructType(clang_getCursorType(cursor))
                 )
             end match
+          else if typ.kind == CXTypeKind.CXType_Pointer then
+            val ct = constructType(clang_getCursorType(cursor))
+            val fixedCt = ct match
+              case CType.Pointer(CType.Reference(Name.Unnamed)) =>
```

Comment:
Thanks for taking a crack at this – unfortunately in current state I think this won't work for double and triple pointers.

If you can confirm that this branch fixes ffmpeg bindings for you, we can merge as is, but I will need to think a bit more for a general approach of dealing with unnamed structs and unions, as the current situation is very brittle.